src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,240 @@
+ *
+ * Radio button example HTML
+ *
+ *    <label class="c-form-label" for="radio1">
+ *        <input class="c-form-input" type="radio" id="radio1" name="myRadioBtn" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--radio"></span>
+ *        My Radio Button Label Text
+ *     </label>
+ *
+ * ================================= */
+ $formControl-bgColor                  : $white !default;
+ $formControl-bgColor-disabled         : $color-disabled;
+ $formControl-borderColor              : $color-border;
+ $formControl-color             : $blue !default;
+ $formControl-background-color  : $color-bg !default;
+ $formControl-hover-color       : $blue--dark !default;
+ $formControl-borderColor-hover : $blue !default;
+ $formControl-margin-left       : 0 !default;
+ $formControl-padding-left      : spacing(x4) !default;
+ $formControl-width--wide       : 20px !default;
+ $formControl-width             : 24px !default;
+ $formControl-height--wide      : 20px !default;
+ $formControl-height            : 24px !default;
+ $formControl-border-width      : 1px;
+
+
+ .o-formControl-label {
+     position: relative;
+     display: block;
+     cursor: pointer;
+     user-select: none;
+     padding: spacing();
+     padding-left: $formControl-padding-left;
+
+     &:hover .o-formControl-indicator {
+         border-color: $formControl-borderColor-hover;
+     }
+
+     &:hover .o-formControl-input:disabled ~ .o-formControl-indicator {
+         border-color: $formControl-borderColor;
+         cursor: not-allowed;
+     }
+
+     .o-formControl-input:disabled ~ .o-formControl-indicator {
+        background-color: $formControl-bgColor-disabled;
+     }
+ }
+
+ // Give inline elements some space to the right
+ .o-formControl-label--inline {
+     display: inline-block;
+     padding-right: 2em;
+ }
+
+     // The hidden input
+     .o-formControl-input {
+         position: absolute;
+         opacity: 0;
+         z-index: -1; // Put the input behind the label so it doesn't overlay text
+
+         &:focus {
+             & + .o-formControl-indicator { @extend %u-elementFocus; }
+             & ~ .o-formControl-text { @extend %u-elementFocus; }
+         }
+
+         .no-csstransforms & {
+             position: relative !important;
+             opacity: 1;
+             z-index: 1;
+         }
+     }
+
+     // The new checkbox/radio
+     .o-formControl-indicator {
+         position: absolute;
+         left: 0;
+         display: inline-block;
+         width: $formControl-width;
+         height: $formControl-height;
+         border: $formControl-border-width solid $grey--lightest;
+         background-color: $formControl-bgColor;
+         vertical-align: middle;
+         margin-left: $formControl-margin-left;
+         margin-top: 0;
+         box-shadow: 0 0 0 2px transparent, 0 0 0 0 transparent; // Used to animate from when element is in :focus
+
+         .no-csstransforms & {
+             display: none;
```

Comment:
More of a question, does this element not appear if css transforms are not supported? Could it not fallback to no animation?

Reply:
We might be able to get rid of this altogether now – I think it was a bit of a hack rather than anything else, but now our support levels are up to IE11, we probably don't need this anymore (and any other `no-csstransforms` references)